starter/script.js

```diff
@@ -1 +1,90 @@
+var userName = prompt("Welcome to the exciting and lucrative world of cookie baking. You'll be presented with a series of choices to guide your growth. Pay extra attention to the words in ALL CAPS! Let's start: what is your name?");
+var choiceOne = prompt("Welcome to " + userName + "'s Bakery! Right now, you have 0 COOKIES. Would you like to BAKE more cookies? Or NOT?").toLowerCase();
+
+if (choiceOne === "not") {
+  alert("You bake no cookies! Your bakery has gone out of business. You lose. Why are you even playing this if you don't want to bake? Come back when you're feeling cooperative.");
+}
+else if (choiceOne === "bake") {
+  alert("You bake a cookie! You have 1 COOKIE. This seems inefficient.")
+  var choiceTwo = prompt("You need to speed up this process. You need to buy something to help! Would you like a CURSOR to help you generate COOKIES faster, or a nice GRANDMA to bake more cookies?").toLowerCase();
+
+  var cursorCount = 0;
+
+  while (choiceTwo === "cursor") {
+    if (cursorCount < 10) {
+      var choiceTwo = prompt("You add a CURSOR to your bakery! I'm unsure how it works, but COOKIES are appearing slightly faster. Would you like another CURSOR, or to try out a nice GRANDMA?").toLowerCase();
+      cursorCount++;
+    }
+    else if (cursorCount >= 10 && cursorCount < 20) {
+      var choiceTwo = prompt("Look. The game's not going to progress until you buy a GRANDMA. Quit screwing around and choose one.").toLowerCase();
+      cursorCount++; }
+    else {
+      alert("Okay, fine! You have so many CURSORS you've baked a million cookies. You win! Game over. Come back when you're willing to actually make other choices.")
+      break;
+    }
+  }
+  if (choiceTwo === "grandma") {
+    alert("Now you have a nice GRANDMA. Your cookies per second are significantly higher! Good choice.");
+    var choiceThree = prompt("Now you're really cooking! It's time to take this bakery to the next level. You can afford one of three new cookie-generating buildings. Would you like to invest in a cookie FARM, a cookie MINE, or a cookie FACTORY?").toLowerCase();
+
+    if (choiceThree === "farm") {
+      var choiceFarm = prompt("You've opened a lovely cookie FARM. You're growing fields on fields of delicious cookies! How would you like to develop this farm further? Do you want to grow ORGANIC cookies, or make some real money with an INDUSTRIAL farm?").toLowerCase();
+
+      if (choiceFarm === "organic") {
+        alert("You make a nice living cranking out organic COOKIES. They're not that different from regular COOKIES, but you get to charge twice as much! Congratulations on your successful COOKIE business! YOU WIN!")
+      }
+      else if (choiceFarm === "industrial") {
+        alert("You pump your COOKIES full of pesticides and modify the heck out of their genetics! Before you know it, you're churning out more COOKIES than you could have ever dreamed! Ethics, shmethics. You've become hecka profitable! YOU WIN!");
+      }
+      }
```

Comment:
at this point i'm really just looking for anything... this should be in line with `if (choiceThree === "farm") {`